specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Line 2043
```
  tabulation_result_table: TYPE =
    sequence[[contest, rla_tabulation_outcome]]
```
I see no need to read the county tabulation results file. The file sample they just sent is a real mess.
We should calculate the winners and `contest_margin` values from the already-parsed CVR files.
In terms of checking integrity, the rla_tabulation_outcome is an intermediate result that they didn't include
in the DQ - no sample, no requirement around checking it etc. IIRC.

At least at this point, I think we should probably present the sets of winners that we calculate when presenting
the contests to the audit board for entry, and later in the generated reports and downloads.